docs/developer-docs/getting-started/default-template.mdx

```diff
@@ -82,22 +82,15 @@ hello/
   hello/
 ├── README.md          # Default project documentation
 ├── dfx.json           # Project configuration file
-├── env.d.ts
-├── jest.config.js
 ├── node_modules
```

Comment:
Should node_modules be listed here? It is not checked in and is generated automatically, similar to the `.azle` and `.dfx` directories.

Reply:
The other examples include `node_modules`, so it is included here for consistency.